### DISCUSSION TITLE
how to upload max filesize:

### DISCUSSION THREAD

**Discussion by Tinoisthe** [SergiX44/XBackBone#428] [Q&A]
how to upload max filesize from 2M to 1 gb I'm using the zip file is there a way ?

**Comment by ahunt0** [comment]
If you haven't solved this already, here's the solution:

You need to modify your php.ini, find the lines that include _[upload_max_filesize](https://www.php.net/manual/en/ini.core.php#ini.upload-max-filesize)_ and _[post_max_size](https://www.php.net/manual/en/ini.core.php#ini.post-max-size)_ change them as desired.

**Reply by Tinoisthe** [reply]
have done this but will not upload after 50 mb

**Reply by ahunt0** [reply]
Is the _post_max_size_ larger than _upload_max_filesize_?

**Reply by Tinoisthe** [reply]
yes upload max is 1gb and post max is 2 gb

**Reply by ahunt0** [reply]
What is your _memory_limit_ set to?

**Reply by Tinoisthe** [reply]
128M

**Reply by ahunt0** [reply]
Try setting _memory_limit_ to -1 (meaning no limit) to test if it is that, and if it is, adjust it from there.

**Reply by Tinoisthe** [reply]
ok  thanks

**Reply by Tinoisthe** [reply]
ok does nothing just sit on 100% and doesn't show up on home page like the rest 

20 to 30 works works any higher it stops

**Reply by ahunt0** [reply]
Are you uploading on your site or through ShareX, try ShareX as well. Double check XBackBone on your site under the system tab that _upload_max_filesize_ & _post_max_size_ are both showing the correct settings. Also do you have any error logs?
Hopefully we can pinpoint the issue.

**Reply by Tinoisthe** [reply]
uploading on the site
Max memory usage: 140M was 128M don't think that will change much  can't do -1 site will not load
Max upload size:
post_max_size: 3G
upload_max_filesize: 2G


Edit fix Long ago
